### DISCUSSION TITLE
hello

### DISCUSSION THREAD

**Discussion by Milentiev** [Chia-Network/chia-blockchain#11231] [Support]
Hello, I have a problem, I updated from version 1.2.11 to 1.3.3 and my coins disappeared, tell me what to do

**Comment by Milentiev** [comment]
And transactions from the pool come, but chia does not see them

**Comment by loppefaaret** [comment]
I would suggest updating to 1.3.4 - and you will likely have to manually remove the current wallet database, and let it resync the wallet from 0 - this should fix any inconsistencies in your wallet database.

You have not said what OS you are on, but by default, under your users home folder, in `~/.chia/mainnet/wallet/db/` is where the wallet database is stored - While you have the Chia client closed down, remove the wallet database, and start the client up again.
This should register all your transactions and coins correctly, and display  the correct balance.

**Comment by Milentiev** [comment]
I did it from scratch many times nothing helps


чт, 21 апр. 2022 г. в 10:49, loppefaaret ***@***.***>:

**Reply by loppefaaret** [reply]
ok, do your wallet report being in sync ? 
Are you running wallet only, or full farming client ? 
Is it show some balance, or is all your coins not showing ?

**Comment by Milentiev** [comment]
yes, let's say the farmer's, I put 1.2.11 everything works, everything
works fine, I throw up the old base. but I put a new client and v2
base, it shows only the old sizes, I don’t know what to do anymore


пт, 22 апр. 2022 г. в 12:57, loppefaaret ***@***.***>:
